README.md

```diff
@@ -29,12 +29,22 @@ client = Egauge::Client.new('http://<device_name>.egaug.es')
 # This nil is necessary because the query structure is 'h&n=24'
 client.query(:h => nil, :n => 24, :f => 1522800000)
 ```
-### Helper query
+### Helper queries
 There are helpers to query data for you without having to craft queries. These also return response objects.
 ```ruby
 require 'egauge'
 client = Egauge::Client.new('http://<device_name>.egaug.es')
 client.full_day_kwh
+client.monthly_kwh
```

Comment:
Based on what we were talking about yesterday, I definitely see a case of potentially having a general method `fetch` that takes a `timeframe` and `number` param where `timeframe` can be one of `[:hour, :day, :month]` and `number` can be any integer greater than `0`.